python/lsst/ts/tcpip/constants.py

```diff
@@ -23,17 +23,21 @@
     "DEFAULT_MONITOR_CONNECTION_INTERVAL",
     "LOCALHOST_IPV4",
     "LOCALHOST_IPV6",
+    "LOCALHOST",
     "LOCAL_HOST",
     "TERMINATOR",
 ]
 
 # Default interval between checks if the connection is alive (seconds)
 DEFAULT_MONITOR_CONNECTION_INTERVAL = 0.1
 
-# localhost constants
+# localhost constants for IPV4 and IPV6
 LOCALHOST_IPV4 = "127.0.0.1"
 LOCALHOST_IPV6 = "::1"
-LOCAL_HOST = LOCALHOST_IPV4  # Deprecated
+# The default localhost
+LOCALHOST = LOCALHOST_IPV4
+# Deprecated alias to LOCALHOST
+LOCAL_HOST = LOCALHOST
```

Comment:
Out of curiosity, when do you plan to remove this?

Reply:
Good question. Perhaps for version 2? The problem is that I don't know how to make use raise a warning.